### PR TITLE
add Celebrate script

### PR DESCRIPTION
**Pull request by Fatpandac**
## Description

Set Confetti run times and intervals

## Type of change

- [X] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

image

## Dependencies / Requirements

Raycast >= 1.35.0

## Checklist

- [X] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/raycast/celebrate.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Celebrate
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon 🎉
+# @raycast.argument1 { "type": "text", "placeholder": "times, defualt by 1", "optional": true }
+# @raycast.argument2 { "type": "text", "placeholder": "interval, defualt by 0", "optional": true  }
+# @raycast.packageName Celebrate
```
Comment (unnamedd) [inline commands/apps/raycast/celebrate.sh:12]:
```suggestion
# @raycast.packageName Raycast
```

File: commands/apps/raycast/celebrate.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Celebrate
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon 🎉
+# @raycast.argument1 { "type": "text", "placeholder": "times, defualt by 1", "optional": true }
```
Comment (unnamedd) [inline commands/apps/raycast/celebrate.sh:10]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Times (Default: 1)", "optional": true }
```

File: commands/apps/raycast/celebrate.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# @raycast.argument2 { "type": "text", "placeholder": "interval, defualt by 0", "optional": true  }
```
Comment (unnamedd) [inline commands/apps/raycast/celebrate.sh:11]:
```suggestion
# @raycast.argument2 { "type": "text", "placeholder": "Interval (Default: 0)", "optional": true  }
```

File: commands/apps/raycast/celebrate.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# @raycast.packageName Celebrate
+
+# Documentation:
+# @raycast.description Set Confetti run times and intervals
```
Comment (unnamedd) [inline commands/apps/raycast/celebrate.sh:15]:
```suggestion
# @raycast.description Set Confetti to run for a number of times and during intervals
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hello @Fatpandac 👋

I've made some comments in your PR in order to keep it conformant to our repository. As soon as you finish to address them, please re-request a review! 

Thanks for your contribution! ;)

**Comment by dehesa** [comment]
This is great @Fatpandac 😄

**Pull request merged**